encoder-stackdriver-brave/src/main/java/zipkin2/reporter/stackdriver/brave/SpanTranslator.java

```diff
@@ -79,6 +91,16 @@ Span.Builder translate(Span.Builder spanBuilder, MutableSpan braveSpan) {
     }
     spanBuilder.setAttributes(attributesExtractor.extract(braveSpan));
 
+    // Spring 6 HTTP spans need mapping to Stackdriver conventional attribute names
+    if (braveSpan.name() != null && braveSpan.name().contains("http")) {
+      braveSpan.tags().forEach((key, value) -> {
+        if (SPRING6_RENAMED_HTTP_LABELS.containsKey(key)) {
+          spanBuilder.getAttributesBuilder()
+                  .putAttributeMap(SPRING6_RENAMED_HTTP_LABELS.get(key), toAttributeValue(value));
+        }
+      });
```

Comment:
I think I would just do a for loop here instead of the `.stream().forEach(...)` shortcut.
If you want to keep the Stream API, I think it would be a bit more natural to move the check to a `.filter(...)`.

Reply:
It's not the Stream API here. This is the `forEach` method on `Map` because `braveSpan.tags()` returns a `Map<String, String>`.